### DISCUSSION TITLE
rust wrapper without cmake for cross compilation

### DISCUSSION THREAD

**Discussion by schmitch** [zxing-cpp/zxing-cpp#729] [Show and tell]
hi I wanted to show our rust wrapper that is based on the wrapper inside this repo but does not use cmake, we created that since we have problems when crossbuilding with cmake: https://github.com/envisia/zxing-rs-no-cmake

might be useful for other people

**Comment by axxel** [comment]
Nice! But please let us continue the discussion [here](https://github.com/zxing-cpp/zxing-cpp/discussions/696#discussioncomment-8493663).
